source/includes/gho/_api.md

```diff
@@ -0,0 +1,305 @@
+
+## EmployeeEdge
+An edge in a connection.
+
+Field | Type | Description
+--------- | ----------- | -----------
+cursor | [String](#string) | A cursor for use in pagination.
+node | [Employee](#employee) | The item at the end of the edge.
+
+## File
+A File record
+
+Field | Type | Description
+--------- | ----------- | -----------
+expires_at | [DateTime](#datetime) | The time when the URL will expire.  After this time, you will need to generate a new URL.
+file_name | [String](#string) | The original name of the file.
+file_size | [Int](#int) | The file size, in bytes
+file_url | [String](#string) | An expiring URL you can use to download the file.
+
+## Location
+    Represents a single location in your company.  Employees can belong only belong to a single Location.  It is    possible for employees to not belong to any Location.  Lepartments are used in a variety of ways in Greenhouse    Onboarding, including permissions and onboarding plans  
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+description | [String](#string) | 
+id | [ID](#id) | 
+name | [String](#string) | 
+updated_at | [DateTime](#datetime) | 
+
+## Mutation
+
+
+Field | Type | Description
+--------- | ----------- | -----------
+updateEmployeeProfile | [Employee](#employee) | Update an Employee's profile
+
+## PageInfo
+Information about pagination in a connection.
+
+Field | Type | Description
+--------- | ----------- | -----------
+endCursor | [String](#string) | When paginating forwards, the cursor to continue.
+hasNextPage | [Boolean](#boolean) | When paginating forwards, are there more items?
+hasPreviousPage | [Boolean](#boolean) | When paginating backwards, are there more items?
+startCursor | [String](#string) | When paginating backwards, the cursor to continue.
+
+## SignatureRequest
+An E-Signature Request for an Employee
+
+Field | Type | Description
+--------- | ----------- | -----------
+counter_signer | [Employee](#employee) | The employee responsible for counter-signing the document, if applicable
+created_at | [DateTime](#datetime) | 
+file | [File](#file) | This is available only for completed signatures.
+id | [ID](#id) | 
+signature_request_template_id | [ID](#id) | 
+status | [SignatureRequestStatus](#signaturerequeststatus) | This request's current status.
+updated_at | [DateTime](#datetime) | 
+
+# Input Objects
+## DateFilter
+Specify a range of dates using after (exclusive >), before (exclusive <), or on (exact match)
+
+Argument | Type | Description
+--------- | ----------- | -----------
+after | [Date](#date) | 
+before | [Date](#date) | 
+on | [Date](#date) | 
+
+## DateTimeFilter
+Specify a range of datetimes using after (exclusive >), before (exclusive <), or on (exact match)
+
+Argument | Type | Description
+--------- | ----------- | -----------
+after | [DateTime](#datetime) | 
+before | [DateTime](#datetime) | 
+on | [DateTime](#datetime) | 
+
+## UpdateCustomFieldValue
```

Comment:
Do we want to enumerate the validation rules here? 😱

Reply:
Which validation rules?  The ones where you always have to have a name and an email?

Reply:
theres a bunch for custom field values depending on the CustomField field_type.

e.g. `custom_field_values` of type contact/address/etc need to be JSON encoded strings with escaped strings

contact values need first_name, last_name, and relationship.

Team CustomFieldvalues require a team value of the same TeamCategory

it goes on and on.

We can chat IRL but do you want me to get a list of these rules together?